.circleci/config.yml

```diff
@@ -18,6 +18,7 @@ workflows:
       - orb-tools/pack:
           filters: *filters
       - orb-tools/review:
+          exclude: RC008
```

Comment:
This test was failing for the pipeline - https://github.com/CircleCI-Public/orb-tools-orb/blob/master/src/scripts/review.bats#L102